deps/catch/single_include/catch2/catch.hpp

```diff
@@ -10732,7 +10732,9 @@ namespace Catch {
 
     // 32kb for the alternate stack seems to be sufficient. However, this value
     // is experimentally determined, so that's not guaranteed.
-    static constexpr std::size_t sigStackSize = 32768 >= MINSIGSTKSZ ? 32768 : MINSIGSTKSZ;
+    // NOTE(aleix): MINSIGSTKSZ is not a constant expresion so compilation
+    // fails. Commeting out for now.
```

Comment:
I think I've seen this issue before, and it was totally an environment thing (LLVM pre-14, maybe?). May be worth taking a second look.

Reply:
Oh, yes. I reverted the change. But I added a few fixes on catch.hpp (copied form mainstrean) to be able to build tests on M1.

Reply:
Actually, the `MINSIGSTKSZ` change is necessary to be able to build tests (only on Linux though). I have investigated and this was due to a change in glibc. I've checked `catch2` upstream and they don't use that code anymore.

In any case, for now I have removed my change.